source/tutorials/tutorial.md

```diff
@@ -323,13 +323,6 @@ When accessing the Builder's Hut block by right-clicking on it, you will see a G
   <div class="col-sm-12 col-md">
     <img src="../../assets/images/gui/buildergui1.png" class="img-fluid mx-auto" alt="Builder's Hut GUI">
   </div>
-  <div class="col-sm-12 col-md">
```

Comment:
Don't delete these, this is how we get the builder's hut GUI from https://github.com/ldtteam/MinecoloniesWiki/blob/master/_data/gui.yml

Reply:
Ah! Okey doke!